lib/urls.ts

```diff
@@ -0,0 +1,9 @@
+const urls = {
+    beingAtTorchbox: '/',
```

Comment:
**Thought** I wonder if `careers` would be a better name, I'm happy with either though.

Reply:
It's a bit awkward as this is the page title, careers does make more sense in the context of the URL